subhub/app.py

```diff
@@ -16,7 +16,7 @@ def create_app(config=None):
     if not CFG("AWS_EXECUTION_ENV", None):
         options = {"swagger_ui": True}
         region = "localhost"
-        host = "http://localhost:8000"
+        host = CFG("DBD_HOST", "http://localhost:8000")
```

Comment:
Oh yeah, and I had to tweak this port because it turns out all the usual ports on my machine are crowded thanks to the FxA stack consisting of 20 microservices or so already.

Reply:
@lmorchard just FYI we are reviewing some options on our end for making this easier.